service-python/assessclaimdc7101/src/lib/bp_calculator.py

```diff
@@ -52,14 +53,20 @@ def bp_reader(request_body):
             not_sortable_bp.append(reading)
             reading["dateFormatted"] = ''
             continue  # If there is no date associated
+
+        if reading["systolic"]["value"] == 0 or reading["diastolic"]["value"] == 0:
+            if bp_reading_date >= date_of_claim_date - relativedelta(years=2):
+                partial_bp_two_years.append(reading)  # to be displayed in PDF
+            continue
```

Comment:
This `continue` ensures we are not counting incomplete readings for assessing sufficiency.